src/components/ProfileInput.jsx

```diff
@@ -0,0 +1,109 @@
+  const [major, setMajor] = React.useState('');
+  const [gender, setGender] = React.useState('');
+  const [daca, setDaca] = React.useState('');
+
+  const handleGradeChange = (event) => {
+    setGrade(event.target.value);
+  };
+  const handleGradChange = (event) => {
+    setGradeYear(event.target.value);
+  };
+  const handleMajorChange = (event) => {
+    setMajor(event.target.value);
+  };
+  const handleGenderChange = (event) => {
+    setGender(event.target.value);
+  };
+  const handleDacaChange = (event) => {
+    setDaca(event.target.value);
+  };
+
+  return (
+    <div>
+      <FormControl className={classes.formControl}>
+        <InputLabel>Grade</InputLabel>
+        <Select labelId="demo-simple-select-label" id="demo-simple-select" value={grade} onChange={handleGradeChange}>
+          <MenuItem value={20}>Highschool Senior</MenuItem>
+          <MenuItem value={30}>College Freshman</MenuItem>
+          <MenuItem value={40}>College Sophomore</MenuItem>
+          <MenuItem value={50}>College Junior</MenuItem>
+          <MenuItem value={60}>College Senior</MenuItem>
+        </Select>
+      </FormControl>
+      <br />
+      <FormControl className={classes.formControl}>
+        <InputLabel>Graduation Year</InputLabel>
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          value={gradYear}
+          onChange={handleGradChange}
+        >
+          <MenuItem value={10}>2020</MenuItem>
+          <MenuItem value={20}>2021</MenuItem>
+          <MenuItem value={30}>2022</MenuItem>
+        </Select>
+      </FormControl>
+      <br />
+      <FormControl className={classes.formControl}>
+        <InputLabel>Major</InputLabel>
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          value={major}
+          onChange={handleMajorChange}
+        >
+          <MenuItem value={10}>Computer Science</MenuItem>
+          <MenuItem value={20}>Chemical Engineering</MenuItem>
+          <MenuItem value={30}>Data Science</MenuItem>
+        </Select>
+      </FormControl>
+      <br />
+      <FormControl className={classes.formControl}>
+        <InputLabel>Gender</InputLabel>
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          value={gender}
+          onChange={handleGenderChange}
+        >
+          <MenuItem value={10}>Male</MenuItem>
+          <MenuItem value={20}>Female</MenuItem>
+        </Select>
+      </FormControl>
+      <br />
+      <FormControl component="fieldset">
+        <FormLabel component="legend">Status</FormLabel>
+        <RadioGroup aria-label="gender" name="gender1" value={daca} onChange={handleDacaChange}>
+          <FormControlLabel value="daca" control={<Radio />} label="Have DACA" />
+          <FormControlLabel value="noDaca" control={<Radio />} label="No DACA" />
+        </RadioGroup>
+      </FormControl>
```

Comment:
For different types of `<FromControl>`s different components can be created as well.

Reply:
So for example a `FormControl` for `Gender`, `SchoolYear` can be simple components that only contain the `FormControl`? Should these be within a folder in the `Components`, as it'll be around 5-6 new components ?